install-ansible.sh

```diff
@@ -42,6 +42,7 @@ if [[ "Debian" =~ $os_VENDOR ]]; then
   pip install PyYAML jinja2 paramiko
   git clone https://github.com/ansible/ansible.git
   cd ansible
+  git submodule update --init --recursive
```

Comment:
That's correct - I was running into this: https://github.com/leucos/ansible-tuto/issues/92